README.md

```diff
@@ -446,6 +454,28 @@ Found '/path/to/project/.nvmrc' with version <5.9>
 Now using node v5.9.1 (npm v3.7.3)
 ```
 
+`.nvmrc` files may have comments. The following is the notation to have a comment inside a `.nvmrc` file:
```

Comment:
This is a pretty major feature that I'm pretty convinced I do not want. In the future it might be best to discuss this sort of thing via issue first, so you don't end up doing wasted work.